docs/lib/vega-lite.md

```diff
@@ -25,33 +25,17 @@ Or you can use a [Vega-Lite JSON view specification](https://vega.github.io/vega
 ```js echo
 vl.render({
   spec: {
-    "width": 640,
-    "height": 400,
-    "data": {
-      "url": await FileAttachment("gistemp.csv").url()
-    },
-    "mark": "point",
-    "encoding": {
-      "x": {
-        "type": "temporal",
-        "field": "Date"
-      },
-      "y": {
-        "type": "quantitative",
-        "field": "Anomaly"
-      },
-      "color": {
-        "type": "quantitative",
-        "field": "Anomaly",
-        "scale": {
-          "range": "diverging",
-          "reverse": true
-        }
-      }
+    width: 640,
+    height: 400,
+    data: {url: await FileAttachment("gistemp.csv").url(), format: {type: "csv"}},
```

Comment:
This is needed because Vega-Lite infers the format type from the file extension (`.csv`), but doesn’t check for the query string and so is confused by the now-present `?sha=…`. It’s better to be explicit about the format anyway.